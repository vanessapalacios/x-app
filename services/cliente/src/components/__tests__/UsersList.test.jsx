import React from 'react';
import { shallow } from 'enzyme';

import UsersList from '../UsersList';
import renderer from 'react-test-renderer';

const users = [
    {
        'active': true,
        'email': 'fiorellapalacios@gmail.com',
        'id': 1,
        'username': 'fiorella'
    },
    {
        'activa': true,
        'email': 'fiorellapalacios@upeu.edu.pe',
        'id': 2,
        'username': 'vanessa'
    }
];

test('UsersList renders properly', () => {
    const wrapper = shallow(<UsersList users={users}/>);
    const element = wrapper.find('h4')
    expect(element.length).toBe(2);
    expect(element.get(0).props.children).toBe('fiorella');
});

test('UsersList renders properly', () => {
    const tree = renderer.create(<UsersList users={users}/>).toJSON();
    expect(tree).toMatchSnapshot();
});