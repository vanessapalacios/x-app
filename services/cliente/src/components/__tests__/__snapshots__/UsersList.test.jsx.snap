// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UsersList renders properly 1`] = `
<div>
  <h4
    className="box litle is-4"
  >
    fiorella
  </h4>
  <h4
    className="box litle is-4"
  >
    vanessa
  </h4>
</div>
`;
